kalite/coachreports/api_views.py

```diff
@@ -107,6 +123,8 @@ def learner_logs(request):
         output_objects.extend(objects)
         output_logs.extend(log_objects)
 
+    output_objects = unique_by_id_and_kind_sort(output_objects)
```

Comment:
I'm a little confused, but I think this kind of question could be cleared up by a few simple examples/client tests of this endpoint. Why does it *matter* that output_objects are sorted and unique?

Also, it seems that `output_objects` is a dictionary mapping topic ids to some value, but both `output_objects` and the key in the JsonResponse below (`"contents"`) are pretty vague.